cmd/utils.go

```diff
@@ -1018,9 +1018,9 @@ func decodeDirObject(object string) string {
 
 // This is used by metrics to show the number of failed RPC calls
 // between internodes
-func loadAndResetRPCNetworkErrsCounter() uint64 {
-	defer rest.ResetNetworkErrsCounter()
-	return rest.GetNetworkErrsCounter()
+func loadAndResetRPCStats() rest.RPCStats {
+	defer rest.ResetRPCStats()
```

Comment:
As discussed we should remove the reset()

Reply:
Done